<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="webWidget" name="viewer" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDockWidget" name="dockWidget">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>462</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>524287</height>
      </size>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget_Controls">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_basicControls">
          <attribute name="title">
           <string>Basics</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_Navigation">
             <property name="title">
              <string>Navigation</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="2">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_places">
                <property name="currentText">
                 <string>Default</string>
                </property>
                <item>
                 <property name="text">
                  <string>Default</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>SVG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>WebGL</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_collab">
          <attribute name="title">
           <string>Collab</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_loadColVizCfgFile">
             <property name="text">
              <string>Load cfg.</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_letsInteract">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Lets Interact</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_ColVizInteract">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Interact</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_sendSelection">
                <property name="text">
                 <string>Send Sel.</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit_textToSend">
                <property name="maximumSize">
                 <size>
                  <width>140</width>
                  <height>100</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_sendMessage">
                <property name="text">
                 <string>Send Mess.</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_persistence">
          <attribute name="title">
           <string>Persistence</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="pushButton_loadPersistence">
             <property name="text">
              <string>Load Persisntece cfg.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_logIn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>LogIn</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_user">
                <property name="text">
                 <string>User</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_user"/>
              </item>
              <item>
               <widget class="QLabel" name="label_password">
                <property name="text">
                 <string>PassWord</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_password"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_logIn">
                <property name="text">
                 <string>logIn</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_LogOut">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Log Out</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_session">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Session</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label_session">
                <property name="text">
                 <string>Session</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_sessioId"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_restoreSession">
                <property name="text">
                 <string>Restore Session</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_chat">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Chat</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_chat"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_url">
     <property name="text">
      <string>https://www.humanbrainproject.eu/en/</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>webWidget</class>
   <extends>QWidget</extends>
   <header>webWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
